docs/reference/PackageReference-with-packages-lock-file.md

```diff
@@ -0,0 +1,66 @@
+---
+title: PackageReference with packages lock file
+description: Details on how to use packages.lock.json file with PackageReference by NuGet 4.9+ and VS2017 and .NET Core 2.1.5xx
+author: jainaashish
+ms.author: jainaashish
+ms.date: 10/18/2018
+ms.topic: conceptual
+---
+
+# Package references (PackageReference) with packages lock file (packages.lock.json)
+
+Package references, using the `PackageReference` node, manage NuGet dependencies directly within project files (as opposed to a separate `packages.config` file) [PackageReference in Project file](https://docs.microsoft.com/en-us/nuget/consume-packages/package-references-in-project-files). 
+
+With PackageReference, Developers do not have confidence that NuGet will restore to the same full closure of package dependencies when they build it on Dev machine vs. CI/CD machines. They also doesn't know anything about their transitive dependencies coming through direct dependencies and when they get changed? (See for more details [Enable repeatable package restore using lock file](https://github.com/NuGet/Home/wiki/Enable-repeatable-package-restore-using-lock-file).)
```

Comment:
The second sentence has at least one typo, but also doesn't read very well to me.

Reply:
which typo mistake? and will try to more simplified. I've already given spec link there to get more details.